conf/routes

```diff
@@ -16,15 +16,6 @@ POST   /api/v1/auth/signUp                  @controllers.AuthenticationControlle
 POST   /api/v1/auth/verifyEmail             @controllers.AuthenticationController.verifyEmail
 POST   /api/v1/auth/resendEmailVerification @controllers.AuthenticationController.resendEmailVerification
 
-# API - Individual Health Data Record
```

Comment:
There are probably integration tests to remove with the removal of this functionality.